deployment/CP04OSPM_Deploy.csv

```diff
@@ -84,13 +84,13 @@ AR-31,,AR-34,,CP04OSPM-WFP01-04-FLORTK000,10,1,2018-11-06T14:36:00,2019-04-22T16
 AR-34,,AR-39,,CP04OSPM-SBS01-00-STCENG000,11,1,2019-04-22T14:13:00,2019-10-13T16:00:00,CGMCP-04OSPM-00011,,CGCON-BSTC01-00004,39.9435,-70.8862,,0,435,
-AR-34,,AR-39,,CP04OSPM-SBS11-02-MOPAK0000,11,1,2019-04-22T14:13:00,2019-10-13T16:00:00,CGMCP-04OSPM-00011,,CGINS-MOPAK0-58484,39.9435,-70.8862,,0,435,sn not on config
-AR-34,,AR-39,,CP04OSPM-WFP01-00-WFPENG000,11,1,2019-04-22T14:13:00,2019-10-13T16:00:00,CGMCP-04OSPM-00011,CGVEH-CWFPCP-10303,CGCON-CWFPCP-10303,39.9435,-70.8862,,416,435,
-AR-34,,AR-39,,CP04OSPM-WFP01-01-VEL3DK000,11,1,2019-04-22T14:13:00,2019-10-13T16:00:00,CGMCP-04OSPM-00011,CGVEH-CWFPCP-10303,CGINS-VEL3DK-00036,39.9435,-70.8862,,416,435,
-AR-34,,AR-39,,CP04OSPM-WFP01-02-DOFSTK000,11,1,2019-04-22T14:13:00,2019-10-13T16:00:00,CGMCP-04OSPM-00011,CGVEH-CWFPCP-10303,CGINS-DOFSTK-02725,39.9435,-70.8862,,416,435,
-AR-34,,AR-39,,CP04OSPM-WFP01-03-CTDPFK000,11,1,2019-04-22T14:13:00,2019-10-13T16:00:00,CGMCP-04OSPM-00011,CGVEH-CWFPCP-10303,CGINS-CTDPFK-00126,39.9435,-70.8862,,416,435,
-AR-34,,AR-39,,CP04OSPM-WFP01-04-FLORTK000,11,1,2019-04-22T14:13:00,2019-10-13T16:00:00,CGMCP-04OSPM-00011,CGVEH-CWFPCP-10303,CGINS-FLORTK-01118,39.9435,-70.8862,,416,435,
-AR-34,,AR-39,,CP04OSPM-WFP01-05-PARADK000,11,1,2019-04-22T14:13:00,2019-10-13T16:00:00,CGMCP-04OSPM-00011,CGVEH-CWFPCP-10303,CGINS-PARADK-20468,39.9435,-70.8862,,416,435,
+AR-34,,AR-39,,CP04OSPM-SBS11-02-MOPAK0000,11,1,2019-04-22T14:13:00,2019-10-13T16:00:00,CGMCP-04OSPM-00011,,CGINS-MOPAK0-11357,39.9435,-70.8862,,0,435,
+AR-34,,AR-39,,CP04OSPM-WFP01-00-WFPENG000,11,1,2019-04-22T14:13:00,2019-10-13T16:00:00,CGMCP-04OSPM-00011,CGVEH-CWFPCP-10303,CGCON-CWFPCP-10303,39.9435,-70.8862,,414,435,
+AR-34,,AR-39,,CP04OSPM-WFP01-01-VEL3DK000,11,1,2019-04-22T14:13:00,2019-10-13T16:00:00,CGMCP-04OSPM-00011,CGVEH-CWFPCP-10303,CGINS-VEL3DK-00036,39.9435,-70.8862,,414,435,
+AR-34,,AR-39,,CP04OSPM-WFP01-02-DOFSTK000,11,1,2019-04-22T14:13:00,2019-10-13T16:00:00,CGMCP-04OSPM-00011,CGVEH-CWFPCP-10303,CGINS-DOFSTK-02725,39.9435,-70.8862,,414,435,
+AR-34,,AR-39,,CP04OSPM-WFP01-03-CTDPFK000,11,1,2019-04-22T14:13:00,2019-10-13T16:00:00,CGMCP-04OSPM-00011,CGVEH-CWFPCP-10303,CGINS-CTDPFK-00126,39.9435,-70.8862,,414,435,
+AR-34,,AR-39,,CP04OSPM-WFP01-04-FLORTK000,11,1,2019-04-22T14:13:00,2019-10-13T16:00:00,CGMCP-04OSPM-00011,CGVEH-CWFPCP-10303,CGINS-FLORTK-01118,39.9435,-70.8862,,414,435,
+AR-34,,AR-39,,CP04OSPM-WFP01-05-PARADK000,11,1,2019-04-22T14:13:00,2019-10-13T16:00:00,CGMCP-04OSPM-00011,CGVEH-CWFPCP-10303,CGINS-PARADK-20468,39.9435,-70.8862,,414,435,
 AR-39,,,,CP04OSPM-RII01-02-ADCPSL010,12,1,2019-10-13T13:42:00,,CGMCP-04OSPM-00012,,CGINS-ADCPSL-21393,39.935558,-70.880328,,406,451,
 AR-39,,,,CP04OSPM-SBS01-00-STCENG000,12,1,2019-10-13T13:42:00,,CGMCP-04OSPM-00012,,CGCON-BSTC01-00012,39.935558,-70.880328,,0,451,
 AR-39,,,,CP04OSPM-SBS11-02-MOPAK0000,12,1,2019-10-13T13:42:00,,CGMCP-04OSPM-00012,,CGINS-MOPAK0-11347,39.935558,-70.880328,,0,451,
```

Comment:
One discrepancy to address for OSPM deployment 5 regarding the MOPAK UID that the csv was changed to.  See specific comment attached to that line in PR.

Reply:
Addressed by SNW and this PR looks good to me - approved.

Reply:
ok